AmazonChimeSDKMachineLearning-Bitcode.podspec

```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |s|
+  s.name             = 'AmazonChimeSDKMachineLearning-Bitcode'
+  s.version          = '0.1.0'
+  s.summary          = 'Amazon Chime SDK Machine Learning for iOS with Bitcode support.'
```

Comment:
Not a must. But is there a better summary to illustrate what is this pod used for?

Reply:
Hmm I updated the summary. Does that work?